meta/service.go

```diff
@@ -0,0 +1,60 @@
+package meta
+
+import (
+	"io/ioutil"
+	"net/http"
+
+	"github.com/hashicorp/raft"
```

Comment:
In Go forums they talk about  gb, wgo, godep. I guess the best thing is to check them all and decide which is best.